src/QuadForm/Herm/Genus.jl

```diff
@@ -18,50 +18,64 @@ local_genus_herm_type(E) = HermLocalGenus{typeof(E), ideal_type(order_type(base_
 ################################################################################
 
 function Base.show(io::IO, ::MIME"text/plain", G::HermLocalGenus)
-  compact = get(io, :compact, false)
-  if !compact
-    if is_dyadic(G) && is_ramified(G)
-       print(io, "Local genus symbol (scale, rank, det, norm) at ")
-    else
-      print(io, "Local genus symbol (scale, rank, det) at ")
-    end
-    if length(G) > 0
-      print(IOContext(io, :compact => true), prime(G), ":")
-      print(io, "\n")
-    else
-      print(IOContext(io, :compact => true), prime(G), " of rank zero")
-    end
+  p = prime(G)
+  println(io, "Local genus symbol for hermitian lattices")
+  nio = pretty(io)
+  io = pretty(io)
```

Comment:
Why is this necessary?

Reply:
Glad you bring this up! The `Lowercase()` is global on your `IOContext`. So for everywhere else you use an `IO` on which you have put `Lowercase()`, the `lowercase_first` is applied, resulting to non-expected lowercasing. For instance, I want to lowercase the first letter for the maximal order, and so I call `Lowercase()` for the `nio` in line 25. But for my printing of `Jordan block`, I want to keep the uppercase: if I use the same `nio`, everything is going to be lowercased, this is why I produce a "different" pretty `IO` to avoid that issue. I hope it seems clear.